.gitignore

```diff
@@ -99,6 +99,8 @@ tags
 /test/**/compperfdat
 /test/**/perfdat
 
+testDatetime
```

Comment:
@leekillough / @daviditen : It doesn't seem to me that this change should be necessary (i.e., we don't list all test names in `.gitignore` (that goodness)).  Was it?

Reply:
I also don't think this line should be added to `.gitignore`.